php/EE/Migration/Executor.php

```diff
@@ -14,13 +14,13 @@ class Executor {
 	public static function execute_migrations() {
 
 		Utils\delem_log( 'ee migration start' );
-		EE::log( 'Migrating EasyEngine data to new version' );
+		EE::debug( "Executing migrations" );
 
 		$migration_paths = self::get_migration_paths();
 
 		if ( empty( $migration_paths ) ) {
-			EE::success( 'Nothing to migrate' );
-			exit( 0 );
+			EE::debug( "Nothing to migrate" );
```

Comment:
Unless string interpolation is required, use single quotes. 

This should have been flagged by PHPCS ideally. If it has not on your machine, ping me tommorow, well set it up.

Reply:
Oh okay. I didn't fix those PHPCS errors intentionally #1124 since this PR of @sagarnasit was already there and I informed him to take care of fixes.